Examples/Scripts/Python/common.py

```diff
@@ -80,9 +80,9 @@ def addPythia8(
 
     NB. this version is included here only for compatibility. Please use pythia8.addPythia8 instead.
```

Comment:
Maybe change the comment to refer to `acts.examples.sumulation`

Reply:
That content is gone form this file.